Assets/Kottaboas/Models/Lekane/Float_bowl.blend.meta

```diff
@@ -0,0 +1,97 @@
+fileFormatVersion: 2
```

Comment:
@HakeemThomas most of my comments are minor, but i think this one's critical.

need to replace the .blend files with .fbx before we can merge it into the game because the .blend files require everyone to have blender installed to open the project. will have to delete these and replace with fbxes.